src/composables/sdk13.ts

```diff
@@ -71,20 +79,16 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
    */
   async function createNodeInstance(url: string) {
     let nodeInstance;
-    isNodeReady.value = false;
-    isNodeError.value = false;
-    isNodeConnecting.value = true;
+    store.commit('setNodeStatus', NODE_STATUS_CONNECTING);
     try {
       // TODO: remove ignore version once HTTP compiler dependency is removed
       nodeInstance = new Node(url, { ignoreVersion: true });
       nodeNetworkId.value = (await nodeInstance.getStatus()).networkId;
-      isNodeReady.value = true;
+      store.commit('setNodeStatus', NODE_STATUS_CONNECTED);
     } catch (error) {
-      isNodeError.value = true;
       nodeNetworkId.value = undefined;
+      store.commit('setNodeStatus', NODE_STATUS_ERROR);
```

Comment:
why can't we store this variable in a composable?

Reply:
I agreed with Subhod that I'll do the refactor for keeping the connection status inside of the composable: https://aeternity-blockchain.atlassian.net/browse/SW-1061

Reply:
Ok, resolving my comment then